src/views/StudyListView.js

```diff
@@ -46,7 +47,7 @@ const StudyListView = ({history}) => {
       </Container>
     );
 
-  if (!loading && !releasesLoading && studyList.length === 0)
+  if (!loading && !hasPermission(myProfile, 'view_study'))
```

Comment:
I think this should be:
```
!hasPermission(myProfile, 'view_study') || !hasPermission(myProfile, 'view_my_study')
```
So that the investigator may still see their own studies.